src/connectors/configure/connectConfigure.ts

```diff
@@ -116,6 +133,7 @@ const connectConfigure: ConfigureConnector = function connectConfigure(
         renderFn(
           {
             refine: connectorState.refine!,
+            setSearchParametersWithoutSearch: connectorState.setSearchParametersWithoutSearch!,
```

Comment:
`refineWithoutApplying`?

Reply:
It's not refining at the moment, so `refineLater` sounds more appropriate than `refineWithoutApplying`. And `draft` sounds like there should be `commit` or `save`, etc.
I struggled the name 😂 and so far I love `refineLater`.

Reply:
What about something like `batchRefine`?

`refineLater` doesn't give a clue as to when it's going to refine.

Reply:
IMO, `batch` feels like there should be a dedicated trigger for that.
Or, what if we use terms like `schedule` or `defer`?
As in `scheduleRender`, ...